plugins/eventMacro/eventMacro/Runner.pm

```diff
@@ -1781,22 +1783,23 @@ sub parse_command {
 			$result = getnpcID($parsed);
 			
 		} elsif ($keyword eq 'cart') {
-			$result = getItemIDs($parsed, $::cart{'inventory'});
+			$result = (getItemIDs($parsed, $char->cart))[0];
 			
 		} elsif ($keyword eq 'Cart') {
-			$result = join ',', getItemIDs($parsed, $::cart{'inventory'});
+			$result = join ',', getItemIDs($parsed, $char->cart);
 			
 		} elsif ($keyword eq 'inventory') {
-			$result = getInventoryIDs($parsed);
+			$result = (getInventoryIDs($parsed))[0];
 			
 		} elsif ($keyword eq 'Inventory') {
 			$result = join ',', getInventoryIDs($parsed);
 			
 		} elsif ($keyword eq 'store') {
+		    # TODO: Fix this to work with the new getItemIDs() implementation.
 			$result = getItemIDs($parsed, \@::storeList);
```

Comment:
Yeah, actually these npc lists should use inventoryList but that is for another day.